xml/Microsoft.Extensions.Logging/LoggingBuilderExtensions.xml

```diff
@@ -126,25 +126,6 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
-    <Member MemberName="Configure">
```

Comment:
This member deletion is the only thing that seems "off" to me - the binaries haven't changed as far as I'm aware. @joelmartinez

Reply:
Hmm ... looking at the changes to the framework index files, it doesn't show this member being deleted there, which suggests it was never actually a part of the manifest that mdoc published. Perhaps this is just an artifact from changes to the .net 5 preview? Seems like an anomaly?

Reply:
@carlossanlop Can you help me figure out why this is getting deleted from the docs? It appears to still be in the source code: https://github.com/dotnet/runtime/blob/a9f6fd69b362f0c4f986bd7bf241bf5ef7d06dce/src/libraries/Microsoft.Extensions.Logging/src/LoggingBuilderExtensions.cs#L58.

Reply:
It's still in the framework index file at https://raw.githubusercontent.com/dotnet/dotnet-api-docs/master/xml/FrameworksIndex/dotnet-plat-ext-5.0.xml:

`<Member Id="M:Microsoft.Extensions.Logging.LoggingBuilderExtensions.Configure(Microsoft.Extensions.Logging.ILoggingBuilder,System.Action{Microsoft.Extensions.Logging.LoggerFactoryOptions})" />`

Reply:
Also tagging @safern for comment.

Reply:
@gewarren ok ... I've created a bug to track this issue:
https://ceapex.visualstudio.com/Engineering/_workitems/edit/298401 

We'll figure out what's happening and include the fix in the next release. In the meantime, you could work around the issue by manually adding this method back into the XML before merging this branch.